nodemon.json

```diff
@@ -4,7 +4,8 @@
         "config.json",
         "config.local.json",
         "browser",
-        "public/styles/theme"
+        "public/styles/theme",
+        "node_modules"
```

Comment:
I don't think it is wise to watch node_modules. Creating watches on each file is expensive and can make development difficult on some machines.

Reply:
Agreed, these are directories/files to ignore watching.  The watch on node_modules was causing be problems.

Reply:
Oh, sorry, my bad.